### ISSUE TITLE
expresshbs is not a function

### ISSUE THREAD

**Issue by chetan-2002** [express-handlebars/express-handlebars#342]
at Object.<anonymous> (E:\Backend Development\Backend-with-express\app.js:14:26)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47

is anyone getting this type or error.

**Comment by chetan-2002** [comment]
I am using express handlebars for the first time. I imported the package from npm. As it is not by default shipped with express so I imported it from node modules in the app.js file.

**const expresshbs = require("express-handlebars");**

Then I set up an engine using the app.engine() method.

**app.engine("handlebars", expresshbs());**

Then I set the view engine of my express app as handlebars.

**app.set("view engine", "handlebars");** 

And when I am running my application it is showing the error: **TypeError: expresshbs is not a function**

I have also attached the complete logs that I am getting in the issue.
